### DISCUSSION TITLE
AdguardHome windows 10/64

### DISCUSSION THREAD

**Discussion by wita74** [AdguardTeam/AdGuardHome#4954] [Q&A]
установлено на windows 10 , перепроверял несколько раз , сам клиент работает но некоторые функции не работают 
1  Белый список не могу создать все время ошибка 
2  Пользовательские правила  ||.......com ^    - в основном не срабатывают в хроме и других браузерах . 
3 Расширение Adguard Browser Extension v3 (отключаю при проверке AgHome ) блокирует на сайте проверке рекламы на 100% и все что добавлю в список , в то время как AgHome блокирует на 79% и что не добавляю все равно не блокирует. В chrome быстрый Dns выключен , со включенным блокирует вообще на 10% .
В общем как настроить и правильно вписать белый список и что делать чтобы правила фильтрации работали ? (все настроено по инструкции перепроверял не раз , фильтры добавлены и тд)

installed on windows 10, rechecked several times, the client itself works but some functions do not work
1 Whitelist can't create all the time error
2 Custom Rules ||.......com ^ - basically don't work in chrome and other browsers.
3 Adguard Browser Extension v3 (disable when checking AgHome) blocks ads on the site by 100% and everything that I add to the list, while AgHome blocks by 79% and doesn’t block anything I don’t add anyway. In chrome, fast Dns is turned off, with it on, it blocks by 10% in general.
In general, how to set up and enter the white list correctly and what should be done to make the filtering rules work? (everything is set up according to the instructions, I checked it more than once, filters were added, etc.)

**Comment by ainar-g** [comment]
>  1 Белый список не могу создать все время ошибка

Пожалуйста, добавляйте конкретный текст ошибки.  Иначе нам сложно понять, что именно не так.

 >  Пользовательские правила ||.......com ^ - в основном не срабатывают в хроме и других браузерах.

Чтобы продиагностировать, нужен пример конкретного правила.

Убедитесь, что опция «Use secure DNS» в Chrome либо отключена, либо направлена на ваш AdGuard Home.  Также убедитесь, что вы проставили как основной, *так и вторичный* адрес DNS-сервера в сетевых настройках.  Иначе часть запросов будет идти в обход AGH.

**Reply by ainar-g** [reply]
(@wita74, лучше отвечать в reply, чтобы дискуссия не расползалась.)

Скриншота не видим, но я подозреваю две концептуальные проблемы:

1.  AdGuard Home является *DNS-блокировщиком.*  То есть, он работает на уровне доменных имён (`crdroid.net`), а не конкретных ресурсов по URL (`https://crdroid.net/downloads`).  Ресурсы можно блокировать в расширении и блокировщике AdGuard.

2.  Соответственно, URL в диалоге добавления исключения — **не** URL ресурса для разблокировки, а URL **списка правил** для доменных имён.  Поэтому ошибка вам и говорит, что `data is HTML, not plain text`.  Ожидается текстовый файл, попадается веб-страница.

Если вы хотите исключить доменное имя `crdroid.net` из DNS-блокировки, самый простой способ это сделать — добавить следующее в *Пользовательские правила фильтрации:*

```none
@@|crdroid.net
```

Это отключит *DNS-фильтрацию* для *доменного имени.*  Фильтрацию же содержимого страницы надо отключать в том, что это содержимое фильтрует.  В вашем случае, как я понял, в расширении.

Почитайте нашу документацию про:

 *  [синтаксис правил DNS-фильтрации](https://adguard-dns.io/kb/ru/general/dns-filtering-syntax/);
 *  [сравнение с сетевой фильтрацией](https://adguard-dns.io/kb/ru/general/dns-filtering/#%D1%81%D1%80%D0%B0%D0%B2%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5-dns-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B8-%D1%81-%D1%81%D0%B5%D1%82%D0%B5%D0%B2%D0%BE%D0%B9-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B5%D0%B9).

**Reply by wita74** [reply]
Добавлял еще до вашего ответа - не работает . Проблема в том что какие то работают правила какие то нет.  У меня внесено @@|crdroid.net^   в пользовательские правила и это не работает
Почему то не могу сюда добавить скриншоты - пишут ошибка

**Reply by wita74** [reply]
либо направлена ​​на ваш AdGuard Home.  - Это как? у меня в хроме можно добавить пользовательские Url адреса это  значить только с http или https  и как я добавлю если у меня только 127.0.0.1 и 192.--.--.251 ? Соответсвенно у меня в хроме это отключено

**Comment by wita74** [comment]
Use secure DNS -отключена 

Также убедитесь, что вы проставили как основной, так и вторичный адрес DNS-сервера в сетевых настройках        -   так все и сделано


конкретно ошибка _ я пытаюсь к примеру добавить сайт  https://crdroid.net/downloads#clover   в белый список и мне выдает ошибку (высылаю скриншот) Хотя вообще ничего не добавляется в исключения , на все выдает ошибку. Информацию о том как именно добавлять и правильно писать не нашел , хотя просят вписать Url (https://crdroid.net/downloads#clover - я вписываю ) и имя ( Crdroid к примеру) жму сохр и ошибка. Может что не так делаю? Или все таки не работает добавление на windows (установлено в самой windows 10 на диске С). Надеюсь на вашу помощь Спасибо
На 26.09.2022 20:07:30, Ainar Garipov ***@***.***> написал(а):
1 Белый список не могу создать все время ошибка
Пожалуйста, добавляйте конкретный текст ошибки. Иначе нам сложно понять, что именно не так.
Пользовательские правила ||.......com ^ - в основном не срабатывают в хроме и других браузерах.
Чтобы продиагностировать, нужен пример конкретного правила.
Убедитесь, что опция «Use secure DNS» в Chrome либо отключена, либо направлена на ваш AdGuard Home. Также убедитесь, что вы проставили как основной, так и вторичный адрес DNS-сервера в сетевых настройках. Иначе часть запросов будет идти в обход AGH.
—
Reply to this email directly, view it on GitHub [https://github.com/AdguardTeam/AdGuardHome/discussions/4954#discussioncomment-3736133], or unsubscribe [https://github.com/notifications/unsubscribe-auth/ABPSS5DETGBKQG52GDPRVMDWAHJ5BANCNFSM6AAAAAAQV46DOM].
You are receiving this because you authored the thread.Message ID: ***@***.***>
[e07956e0-ebb1-499a-9b25-46cb8b838a9f]

**Reply by wita74** [reply]
опция «Использовать безопасный DNS» в Chrome отключена
вы проставили как основной, так и вторичный адрес DNS-сервера в реализации - да все так и было

**Comment by wita74** [comment]
Добавлял в пользовательское как вы и написали ,вот тут тоже кроется проблема- не работает это

⁣Виталий ​

На 26 сент. 2022 г., 20:40, в 20:40, Ainar Garipov ***@***.***> ***@***.***, лучше отвечать в reply, чтобы дискуссия не расползалась.)
>
>Скриншота не видим, но я подозреваю две концептуальные проблемы:
>
>1.  AdGuard Home является *DNS-блокировщиком.*  То есть, он работает на
>уровне доменных имён (`crdroid.net`), а не конкретных ресурсов по URL
>(`https://crdroid.net/downloads`).  Ресурсы можно блокировать в
>расширении и блокировщике AdGuard.
>
>2.  Соответственно, URL в диалоге добавления исключения — **не** URL
>ресурса для разблокировки, а URL **списка правил** для доменных имён. 
>Поэтому ошибка вам и говорит, что `data is HTML, not plain text`. 
>Ожидается текстовый файл, попадается веб-страница.
>
>Если вы хотите исключить доменное имя `crdroid.net` из DNS-блокировки,
>самый простой способ это сделать — добавить следующее в
>*Пользовательские правила фильтрации:*
>
>```none
>@@|crdroid.net
>```
>
>Это отключит *DNS-фильтрацию* для *доменного имени.*  Фильтрацию же
>содержимого страницы надо отключать в том, что это содержимое
>фильтрует.  В вашем случае, как я понял, в расширении.
>
>Почитайте нашу документацию про:
>
>*  [синтаксис правил
>DNS-фильтрации](https://adguard-dns.io/kb/ru/general/dns-filtering-syntax/);
>*  [сравнение с сетевой
>фильтрацией](https://adguard-dns.io/kb/ru/general/dns-filtering/#%D1%81%D1%80%D0%B0%D0%B2%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5-dns-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B8-%D1%81-%D1%81%D0%B5%D1%82%D0%B5%D0%B2%D0%BE%D0%B9-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B5%D0%B9).
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/AdguardTeam/AdGuardHome/discussions/4954#discussioncomment-3736379
>You are receiving this because you were mentioned.
>
>Message ID:
***@***.***>

**Comment by wita74** [comment]
Хочу перед Вами извиниться . Простите меня пожалуйста за мое поведение и если Вас оскорбил как то. Все дело оказалось в расширении  RealProxy которое работает на обход блокировок рунета и на которое я никак не мог подумать. Я отключил Adguard Home и расширение и сайт все равно стал показывать что установлен блокировщик. Тогда я стал отключать расширение по одному и сразу нашел в чем проблема. 
Теперь вроде все пользовательские правила работают
Извините еще раз меня пожалуйста

**Reply by ainar-g** [reply]
Всё нормально, извините за отсутсвие ответов.  Надеемся, что дальше всё будет работать хорошо.

**Comment by wita74** [comment]
Получается : Adguard Home на windows 
1 Не работает белый список
2 Не работают пользовательские правила тк при вводе ||crdroid.com^   (к примеру) это не блокируется или @@||plarium.com^    это тоже не работает после внесения в список пользовательских фильтров тк пишут обнаружен блокировщик
 Выходит не работает на 50% а может и больше 
Зачем тогда добавлять эти функции в приложение на windows
  
На 26.09.2022 20:40:39, Ainar Garipov ***@***.***> написал(а):
***@***.*** [https://github.com/wita74], лучше отвечать в reply, чтобы дискуссия не расползалась.)
Скриншота не видим, но я подозреваю две концептуальные проблемы:
*
AdGuard Home является DNS-блокировщиком. То есть, он работает на уровне доменных имён (crdroid.net), а не конкретных ресурсов по URL (https://crdroid.net/downloads). Ресурсы можно блокировать в расширении и блокировщике AdGuard.
*
Соответственно, URL в диалоге добавления исключения — не URL ресурса для разблокировки, а URL списка правил для доменных имён. Поэтому ошибка вам и говорит, что data is HTML, not plain text. Ожидается текстовый файл, попадается веб-страница.
Если вы хотите исключить доменное имя crdroid.net из DNS-блокировки, самый простой способ это сделать — добавить следующее в Пользовательские правила фильтрации:
@@|crdroid.net
Это отключит DNS-фильтрацию для доменного имени. Фильтрацию же содержимого страницы надо отключать в том, что это содержимое фильтрует. В вашем случае, как я понял, в расширении.
Почитайте нашу документацию про:
* синтаксис правил DNS-фильтрации [https://adguard-dns.io/kb/ru/general/dns-filtering-syntax/];
* сравнение с сетевой фильтрацией [https://adguard-dns.io/kb/ru/general/dns-filtering/#%D1%81%D1%80%D0%B0%D0%B2%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5-dns-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B8-%D1%81-%D1%81%D0%B5%D1%82%D0%B5%D0%B2%D0%BE%D0%B9-%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B5%D0%B9].
—
Reply to this email directly, view it on GitHub [https://github.com/AdguardTeam/AdGuardHome/discussions/4954#discussioncomment-3736379], or unsubscribe [https://github.com/notifications/unsubscribe-auth/ABPSS5ARHV75RVZUOJW4J5DWAHNZLANCNFSM6AAAAAAQV46DOM].
You are receiving this because you were mentioned.Message ID: ***@***.***>
[af8a3e9d-45fa-43fb-9b4b-6467bc5545d6]

**Reply by ainar-g** [reply]
В очередной раз прошу пользоваться интерфейсом GitHub.  Иначе ваши письма открывают новые темы, и тяжело следить за дискуссией.

Ваши проблемы с белым списком воспроизвести не можем.  Добавляем `||google.com` (заметьте, без `@@`, так как это уже белый список), и он разблокируется.

То же с пользовательскими правилами.  Убедитесь, что как ОС, так и браузер настроены на использование AdGuard Home.
